### PR TITLE
Small gramatical fixes and making sentences more exact

### PR DESCRIPTION
**Pull request by loevlie**

### CODE REVIEW

File: simclusters-ann/README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 SimClusters ANN is a service that returns tweet candidate recommendations given a SimClusters embedding. The service implements tweet recommendations based on the Approximate Cosine Similarity algorithm.
 
-The cosine similarity between two Tweet SimClusters Embedding represents the relevance level of two tweets in SimCluster space. The traditional algorithm for calculating cosine similarity is expensive and hard to support by the existing infrastructure. Therefore, the Approximate Cosine Similarity algorithm is introduced to save response time by reducing I/O operations.
+The cosine similarity between two Tweet SimClusters Embeddings represents the relevance level of two tweets in SimCluster space. The traditional algorithm for calculating cosine similarity is expensive and hard to support by the existing infrastructure. Therefore, the Approximate Cosine Similarity algorithm is introduced to save response time by reducing I/O operations.
```
Comment (spewolf) [inline simclusters-ann/README.md:5]: If you make embeddings plural, you need to change represents to represent

**Comment by ifeanyichukwuOtiwa-sports** [comment]
LGTM

File: simclusters-ann/README.md (hunk 1 above)
Comment (loevlie) [reply simclusters-ann/README.md]: True, I have now made that change.

Comment (spewolf) [reply simclusters-ann/README.md]: Thanks @loevlie LGTM 🚢

**Comment by loevlie** [comment]
Please reopen this.

**Pull request closed without merging**